sites/shared/styles/svg-freesewing-draft.css

```diff
@@ -297,11 +297,16 @@ path.gridline.metric.sm {
 path.gridline.metric.xs {
   stroke-dasharray: 1 1;
 }
-path.gridline.imperial {
-  stroke-dasharray: 5 5;
+path.gridline.imperial.md {
+  stroke-width: 0.15 !important;
```

Comment:
Everytine you have to resort to use `!important` in CSS is typically a good time to pause and refect on how to write better selectors